### DISCUSSION TITLE
Rad Pro: History vertical scaling

### DISCUSSION THREAD

**Discussion by PlasmaHH** [Gissio/radpro#116] [Q&A] [answered]
(on the GC-01 )

The vertical autoscaling usually functions fine, but sometimes there is at least one datapoint that pulls things down and then about half the screen is basically lost:

![image](https://github.com/Gissio/radpro/assets/1965659/e4c8ea8d-bc55-4482-af36-b8d2ef39ab4d)

is there a way to control the scaling? Also is there maybe a way to expand the graph (the 1/0.01 could be inside the graph for that) or maybe even switch between log/linear to find the best view of an intresting piece of data?

Furthermore, for the "zoomed out" timespans, is there maybe a way to enable min/avg/max view like this:

![image](https://github.com/Gissio/radpro/assets/1965659/580ae1b5-ffed-4496-94e3-a37a9346fdd8)

**Comment by Gissio** [comment]
When measuring ionizing radiation, usually the order of magnitude (the logarithmic value) matters more than the linear value. That’s why both the history and the bar view of the instantaneous measurement mode are logarithmic (e.g., every division represents a tenfold increase in value). And that’s why a scale which increases tenfold for every division makes more sense.

Also, keep in mind that due to the statistical nature of the pulses, the measurement error is on the order of 50%. So what you are seeing is expected to be noisy.

**Reply by PlasmaHH** [reply]
Yes, I understand the logarithmic nature (like a lot of stuff in physics, especially in human perception) but this is about getting the most details out of it in case one is curious. 

outliers messing up the scaling can be especially annoying (see the first photo) in my experience. Especially in the first place when measuring something I like to zoom in and out and around to get a feel for it.


For the second screenshot: For longer measurements I think its pretty intresting to see not necessarily the exact ones, but if there are outliers or not. If in a 24h log there is an average of .4µSv/h but outliers of 20µSv/h for a minute (or whatever makes statistical sense) then the view shown in the screenshot is very helpful in seeing that there is something worth investigating more.

**Reply by gc01noob** [reply, marked as answer]
might not be helpful but this firmware lets you connect a PC via USB for data capture and analysis and the graphs are far more detailed - you might want to try "geigerlog" at least as an interim measure.
https://github.com/Gissio/geigerlog-radpro

**Reply by PlasmaHH** [reply]
> might not be helpful but this firmware lets you connect a PC via USB for data capture and analysis and the graphs are far more detailed - you might want to try "geigerlog" at least as an interim measure. https://github.com/Gissio/geigerlog-radpro

Yes I know about that feature but that is only really feasible when having a stationary setup. The nice thing about those cheap geiger counters is that you can have them multiple around at various places, moving etc. and then later look at the history.

**Reply by alfmck** [reply]
Would be a good solution to have an option for sending accumulated into device log data to LCD. In this case with radpro we will get a solid measuring device with data registering function.
